core/templates/components/entity-creation-services/exploration-creation.service.ts

```diff
@@ -84,8 +84,9 @@ export class ExplorationCreationService {
       this.loaderService.showLoadingScreen('Creating exploration');
 
       var form = new FormData();
-      form.append('yaml_file', yamlFile);
-      form.append('payload', JSON.stringify({}));
+      form.append('payload', JSON.stringify({
+        yaml_file: yamlFile
```

Comment:
I feel like you might want to tell @Nik-09 (and @rohitkatlaa) about this change. He's been grappling recently with how to handle uploaded files in payloads and the standardization here might be useful for his GSoC project (not sure).

Reply:
Done.